packages/app/src/app/pages/Dashboard/Components/Header/index.tsx

```diff
@@ -87,9 +86,7 @@ export const Header = ({
       align="center"
       justify="space-between"
       css={{
-        width: `calc(100% - ${2 * GUTTER}px)`,
-        maxWidth: GRID_MAX_WIDTH - 2 * GUTTER,
-        margin: '0 auto', // Negative margin top to align visually w/ the sidebar
```

Comment:
This replicates the `VariableGrid` max width calculations, removing these makes the title get misaligned with the content on high-res, ultra-wide screens:

![Screen Shot 2023-02-01 at 22 35 19](https://user-images.githubusercontent.com/24959348/216209441-1c8101ed-2cb9-499b-9689-cc91c39843c2.png)